### PR TITLE
[Keymap] Drashna Code Cleanup

### PR DESCRIPTION
**Pull request by drashna**
## Description

* Add license header to all my files
* Fix issue with pimoroni code
* Add Launchpad keymap
* Cleanup of code

## Types of Changes

- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/launchpad/keymaps/drashna/config.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#pragma once
```
Comment (noroadsleft) [inline keyboards/launchpad/keymaps/drashna/config.h:1]: License header?

File: keyboards/launchpad/keymaps/drashna/keymap.c [hunk 2]
```diff
@@ -0,0 +1,94 @@
+// Below layout is based upon /u/That-Canadian's planck layout
```
Comment (noroadsleft) [inline keyboards/launchpad/keymaps/drashna/keymap.c:1]: License header?

File: keyboards/launchpad/keymaps/drashna/readme.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Launch Pad
```
Comment (noroadsleft) [inline keyboards/launchpad/keymaps/drashna/readme.md:1]: Any reason for the copy of the `default_rgb` keymap's readme here?

File: keyboards/launchpad/keymaps/drashna/config.h (hunk 1 above)
Comment (drashna) [reply keyboards/launchpad/keymaps/drashna/config.h:1]: Curses, I thought I got them all... :D

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**